docs/how-to-guides.md

```diff
@@ -32,6 +32,199 @@ Everything about test data generation and its adjustable parameters can
 be found in the [Generating a Test Dataset with niceML](
 generate-data.md#step-3-customizing-data-generation-optional)
 
+## How to write my own pipeline
+NiceML provides configurable, re-usable machine learning 
+pipelines. NiceML comes with pre-configured pipelines for object detection, 
+semantic semgmentation, regression, classification and others. But what if 
+your problem is not covered (yet) by the pipeline portfolio niceML offers?
+This guide will walk you through how to set up your first custom pipeline.
+The goal of the pipeline is to orchestrate a classifier that can distinguish
+between daisies and roses.
+
+1. Download the images from [Kaggle](https://www.kaggle.com/datasets/alxmamaev/flowers-recognition)
```

Comment:
Sometimes the Steps have a header (e.g. '6. Configure prediction operation') sometimes they don't (e.g. '2. Let us start configuring at the highest level of niceML: The [...]'). Please stay consistent.
I would suggest to use headers. If this section will be moved to the Tutorials, its format should be consistent with the existing tutorials, where subheaders are used with an introducing 'Step X: Header of the step'.

Reply:
I moved the section to tutorials and changed the style accordingly. Could you check the code for me?

Reply:
Done